client/src/SetRating.js

```diff
@@ -0,0 +1,13 @@
+import { useState } from "react";
+
+export default function SetRating({ ratin, setRating, video }) {
```

Comment:
This component doesn't seem to use its `ratin` or `setRating` props? Why does it have them?

Reply:
Right!!
Thanks